suite-native/app/app.config.ts

```diff
@@ -4,28 +4,22 @@ import { ExpoConfig, ConfigContext } from 'expo/config';
 
 import { suiteNativeVersion } from './package.json';
 
-type BuildType = 'debug' | 'develop' | 'staging' | 'production';
+type BuildType = 'debug' | 'develop' | 'production';
 
-const getBuildType = (): BuildType => {
-    if (process.env.ENVIRONMENT === 'production') return 'production';
-    if (process.env.ENVIRONMENT === 'staging') return 'staging';
-    if (process.env.ENVIRONMENT === 'develop') return 'develop';
-    return 'debug';
-};
-const buildType = getBuildType();
+const environment = process.env.ENVIRONMENT as BuildType;
+
+const buildType = environment ?? ('debug' as BuildType);
 
 const bundleIdentifiers = {
     debug: 'io.trezor.suite.debug',
     develop: 'io.trezor.suite.develop',
-    staging: 'io.trezor.suite.staging',
     production: 'io.trezor.suite',
 } as const satisfies Record<BuildType, string>;
-const bundleIdentifier = bundleIdentifiers[buildType];
+const bundleIdentifier = bundleIdentifiers[environment || buildType];
```

Comment:
Do we need both variables: environment and buildType when everywhere we are using both and environment is embedded in the buildType?

`const buildType = (process.env.ENVIRONMENT ?? 'debug') as BuildType;`

Reply:
Good point

https://github.com/trezor/trezor-suite/pull/10898/commits/a04246ef4c716be07a262b518a3a47261878d6d4